core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -95,6 +95,8 @@
     private Runnable responseTimeoutHandler;
     @Nullable
     private TimeoutController responseTimeoutController;
+    @Nullable
+    private Consumer<TimeoutController> pendingTimeoutTask;
```

Comment:
Is there a chance a user calls timeout methods multiple times? Probably better chaining with `andThen()`